cookbooks/bcpc-hadoop/recipes/zabbix_web_monitoring.rb

```diff
@@ -0,0 +1,135 @@
+ruby_block "zabbix_web_monitoring" do
+  block do
+    require 'zabbixapi'
+
+    # Make connection to zabbix api url
+    zbx = ZabbixApi.connect(
+      :url => "https://#{node['bcpc']['management']['vip']}" +
```

Comment:
Perhaps this string could be created in the attributes file in the case of a cluster which has Zabbix running elsewhere?